mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -1152,4 +1170,32 @@ class GeneralizedLinearRegressionTrainingSummary private[regression] (
         "No p-value available for this GeneralizedLinearRegressionModel")
     }
   }
+
+  /**
+   * Summary table with feature name, coefficient, standard error,
+   * tValue and pValue.
+   */
+  @Since("2.2.0")
+  lazy val summaryTable: DataFrame = {
+    if (isNormalSolver) {
+      var featureNames = featureName
+      var coefficients = model.coefficients.toArray
+      var idx = Array.range(0, coefficients.length)
+      if (model.getFitIntercept) {
+        featureNames = featureNames :+ "Intercept"
+        coefficients = coefficients :+ model.intercept
+        // Reorder so that intercept comes first
+        idx = (coefficients.length - 1) +: idx
+      }
+      val result = for (i <- idx.toSeq) yield
+        (featureNames(i), coefficients(i), coefficientStandardErrors(i), tValues(i), pValues(i))
+
+      val spark = SparkSession.builder().getOrCreate()
+      import spark.implicits._
+      result.toDF("Feature", "Estimate", "StdError", "TValue", "PValue").repartition(1)
```

Comment:
question: is "Estimate" the better term to use here as opposed to "Coefficient"?  Are there other libraries which use this specific term in this case?